org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/XMLSchemaCompletionExtensionsTest.java

```diff
@@ -301,6 +301,33 @@ public void completionWithoutStartBracket() throws BadLocationException {
 		
 	}
 
+	@Test
+
+	public void xsiCompletionTest1() throws BadLocationException {
+		String xml = 
+		"<project\r\n" +
+		"    xmlns=\"http://maven.apache.org/POM/4.0.0\"\r\n" +
+		"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" +
+		"    xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\" |>\r\n" +
+		"</project>";
+		XMLAssert.testCompletionFor(xml, false, c("xsi:nil", "xsi:nil=\"\""), c("xsi:type", "xsi:type=\"\""), c("xsi:noNamespaceSchemaLocation", "xsi:noNamespaceSchemaLocation=\"\""));
+	}
+
+	@Test
+
+	public void xsiCompletionTestDoesntAppear() throws BadLocationException {
+		String xml = 
+		"<project\r\n" +
+		"    xmlns=\"http://maven.apache.org/POM/4.0.0\"\r\n" +
+		"    xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\r\n" +
+		"    xsi:schemaLocation=\"http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd\" |>\r\n" +
+		"  <modelVersion xs|></modelVersion>\r\n" +
+		"</project>";
+
+		XMLAssert.testCompletionFor(xml, false);

```

Comment:
It should be XMLAssert.testCompletionFor(xml, 0); -> 0 mean that you expect that you will have none completion items